src/main/external-resources/UMS.conf

```diff
@@ -196,7 +196,7 @@ http_engine_v2 =
 # Prevent system idle sleep
 # -------------------------
 # If your UMS host has a power-saving mode, use this option to prevent the
-# system from hibernating or sleeping. Currently only Windows and macOS are
+# system from hibernating or sleeping. Currently only Windows and Mac OS are
 # supported. There are three modes:
```

Comment:
It is "wrong" in that it isn't the official name. Don't blame me that aPple keeps changing the name, the current name is ```macOS```.